server/hap2/hatohol/hap2_fluentd.py

```diff
@@ -135,7 +149,8 @@ def __parse_line(self, line):
             timestamp, tag = self.__parse_header(header)
         except:
             timestamp, tag, msg = None, None, None
-            haplib.handle_exception()
+            exctype, value = haplib.handle_exception()
```

Comment:
This `exctype` is no longer used?

If so, we should use `_` instead of `exctype` to clarify "no longer used".